eclipse-theia/theia
# Show tabbar when items present

#### What it does


Fixes part of #12305 (but not all)

At the React 18 uplift, an old `onRender` Disposable in the `TabBarToolbar` class was replaced with a new-style `onRender` mechanism, which, however, doesn't appear to work reliably. Simpler is just revealing the toolbar immediately, knowing that a render will come soon.

#### How to test


1. Reveal the Explorer
2. Reload the application
3. More TabbarToolbar item should be visible on startup without changing view containers.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
